src/frameworks/uikit/ui_image.rs

```diff
@@ -7,6 +7,7 @@
 
 use crate::frameworks::core_graphics::cg_image::{self, CGImageRef, CGImageRelease};
 use crate::frameworks::core_graphics::CGSize;
+use crate::frameworks::foundation::ns_data::to_rust_slice;
```

Comment:
Can you change the import to be just `ns_data` (you can merge it with the following line), and then write `ns_data::to_rust_slice` at the callsite? That's what I do for these pseudo-methods on `ns_string`.

Reply:
Done